packages/web/src.ts/types.ts

```diff
@@ -1,5 +1,13 @@
 "use strict";
 
+import * as http from 'http';
+import * as https from 'https';
+
+export interface HttpProviderAgent {
+    http?: http.Agent;
+    https?: https.Agent;
+}
+
```

Comment:
This code is also being used by Web3.js https://github.com/ChainSafe/web3.js/blob/1.x/packages/web3-providers-http/types/index.d.ts#L32 as well as many typescript implementations using `http.Agent`.